source/gatherings/template.html.erb

```diff
@@ -373,5 +391,16 @@ description:  The OpenShift Commons community gets together and share experience
 
     // match height of speaker icons
     $('.speaker-container').matchHeight();
+
+    // grey out and "deactivate" obsolete elements if the event already started
+    var eventDate = new Date("<%= gathering.date.to_time.iso8601() %>");
+    if (Date.now() > eventDate) {
```

Comment:
What do you think about doing something like this, https://stackoverflow.com/a/563442/6758654, so that we consider the event over at 1 day past the `eventDate`? At least this way, the day of, the page still looks active.

Reply:
Checking against the following day sure makes sense. The [preview](https://com-stage-pro.b9ad.pro-us-east-1.openshiftapps.com/gatherings/Helsinki_2018.html) has been updated (no visible change, the threshold day is logged in console).